src/DotVVM.Framework/ViewModel/Validation/ViewModelValidationRuleTranslator.cs

```diff
@@ -34,7 +36,9 @@ public virtual IEnumerable<ViewModelPropertyValidationRule> TranslateValidationR
                         validationRule.ClientRuleName = "range";
                         validationRule.Parameters = new[] { rangeAttr.Minimum, rangeAttr.Maximum };
                         break;
-                    case DotvvmEnforceClientFormatAttribute enforceClientFormatAttr:
+                    case DotvvmClientFormatAttribute enforceClientFormatAttr:
+                        addEnforceClientFormat = false;
+                        if (enforceClientFormatAttr.Disable) yield break;
```

Comment:
Hey, this `yield break;` interrupts the execution and will suppress any following validators.

Reply:
Oh, that's a silly mistake. Sorry about that.